src/pages/index.tsx

```diff
@@ -1,14 +1,19 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui'
 import React from 'react'
-import { NextPage, GetStaticProps } from 'next'
+import { NextPage, GetServerSideProps } from 'next'
 
+import { SiteClient } from 'datocms-client'
 import { Meta, Case } from '../types/content'
 import SEO from '../components/seo'
 import MainLayout from '../layouts/main'
 import Section from '../components/section'
 import Card from '../components/card'
 
+require('dotenv').config()
+
+const client = new SiteClient(process.env.API_KEY)
```

Comment:
an env var should be configured in the server `.env` file

Reply:
as per our conversation, we will be hardcoding this value, since it is a read-only token